valhalla/baldr/tilegetter.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include <valhalla/baldr/curler.h>
+
+#include <string>
+#include <utility>
```

Comment:
i notice a bunch of utility includes. that was the header that is mainly used for std::pair. i cant figure out what this header is needed for

Reply:
it's for `std::move` also

Reply:
oh man that is so odd! i had no idea `move` was in that header!